eth2/types/src/eth_spec.rs

```diff
@@ -0,0 +1,91 @@
+// Copied from file of the same name at https://github.com/sigp/lighthouse
+// with some modifications and deletions. Specifically, removed code that wasn't necessary for this
```

Comment:
@gjtrowbridge do we need all these configurations? It seems like most of this would be irrelevant?

Reply:
I think it's nice to have these--it has the whole spec in there, which is nice for seeing both full context on what values are defined in the spec and because if/when we need them we can use them.  One option that maybe is more attractive is I can comment out the parts that aren't actively in use yet (similar to what I did in BeaconState so folks can see the spec but don't need to worry about the commented out parts).  Open to thoughts here.

Reply:
Yeah, commenting out seems good from my end. It could be helpful for someone to dive in and see easily what is actually relevant in the current state.